<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="1" column="1">
    <widget class="QToolButton" name="button_flip_limits">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0px</string>
     </property>
     <property name="text">
      <string>â‡„</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="value_vmin"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="value_vmax"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="combodata_att"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QColorBox" name="color_color">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSlider" name="value_alpha">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorBox</class>
   <extends>QLabel</extends>
   <header>glue.utils.qt.colors</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
